packages/labs/observers/src/intersection_controller.ts

```diff
@@ -147,6 +148,7 @@ export class IntersectionController<T = unknown> implements ReactiveController {
    * @param target Element to observe
    */
   observe(target: Element) {
+    this._targets.add(target);
```

Comment:
I think the DOM APIs won't observe the same element multiple times, but can you confirm?

Reply:
From what spec text I can find:

- Resize Observer will ensure only one subscription per element, by unobserving and re-observing existing elements. ([ref](http://w3c.github.io/csswg-drafts/resize-observer-1/#dom-resizeobserver-observe))

- Intersection Observer will early return if an element is already being observed. ([ref](https://w3c.github.io/IntersectionObserver/#observe-target-element))

- Mutation Observer is less clear. It looks like it may allow multiple `observe` calls per element because it allows different config options per `observe` call. In our case we always reuse the same config.([ref](https://dom.spec.whatwg.org/#dom-mutationobserver-observe)).

I can add a test for the mutation observer case.

Reply:
I manually tested the mutation case. The same element with the same config will not be observed multiple times. Resolving.